src/extensions/colors/inspector.js

```diff
@@ -62,29 +63,16 @@ function addAttributes( settings ) {
 /**
  * Override the default edit UI to include a new block inspector control
  *
- * @param {Function} BlockEdit Original component.
- * @return {string} Wrapped component.
+ * @param {Object} props Selected block props.
  */
-const withInspectorControl = createHigherOrderComponent( ( BlockEdit ) => {
-	return ( props ) => {
-		const allowedBlocks = [ 'core/list', 'core/quote' ];
-		return (
-			<Fragment>
-				<BlockEdit { ...props } />
-				{ props.isSelected && allowedBlocks.includes( props.name ) && <Inspector { ... { ...props } } /> }
-			</Fragment>
-		);
-	};
-}, 'withInspectorControl' );
+const useColorControls = ( props ) => {
+	const allowedBlocks = [ 'core/list', 'core/quote' ];
+	return (
+		<>
```

Comment:
I don't think the fragment is needed here too.

Reply:
Unfortunately same issue with wrapped JSX conditional I am receiving an unexpected token error. I suspect the better way to handle this would be to return null early instead of the conditional wrap method.

Reply:
Oh, I like the tertiary idea. I'll get this pushed up with a tertiary instead it looks nice.